public/styles/main.css

```diff
@@ -252,6 +252,9 @@ a.connections-table.icon {
     justify-content: space-between;
     min-height: auto;
   }
+  .connections.list.nav {
```

Comment:
can we hyphenate this please instead o fmultiple classes?

Reply:
done and I hyphenated it in the other place it was used in too